src/os/file_other.go

```diff
@@ -78,6 +95,15 @@ func (f stdioFileHandle) Fd() uintptr {
 //go:linkname putchar runtime.putchar
 func putchar(c byte)
 
+//go:linkname getchar runtime.getchar
+func getchar() byte
+
+//go:linkname buffered runtime.buffered
+func buffered() int
+
+//go:linkname gosched runtime.Gosched
```

Comment:
Does this need to be defined here? I thought it already was elsewhere.

Reply:
Defined below, but I believe the following is the definition of `machine.gosched()`.
Since it is not possible to import runtime packages, etc. from os packages, `go:linkname` is used here.
This defines `os.gosched()`.

https://github.com/tinygo-org/tinygo/blob/cc3009b5c15e0772f9f62c3c0ff6789e2bb961cf/src/machine/machine_nxpmk66f18_uart.go#L63-L64